src/workloads/query/GroupLikeDistinct.yml

```diff
@@ -0,0 +1,169 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of $group queries with distinct like semantics, meaning they will
+  execute with a DISTINCT_SCAN. Currently this is supported for $group + $sort and only with the
+  $first and $last accumulators. In PM-3163 this will be extended to $top, $topN and $bottom,
+  $bottomN accumulators so we are including those here in order to see performance changes once that
+  is implemented.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
+
+Keywords:
+- Distinct
+- Group
+- First
+- Last
+- Top
+- Bottom
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 11
+
+ActorTemplates:
+- TemplateName: RunAggCommand
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "RunAggCommand"}}
+    Type: CrudActor
+    Database: *Database
+    Threads: 32
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Duration: 30 seconds
+          Collection: *Collection
+          Operations:
+          - OperationMetricsName: DistinctExecutionMetrics
```

Comment:
How is the `OperationMetricsName` used in the reports? As far as I remember, the collected timeseries will be labeled with `PhaseName.aggregate`, it doesn't help to make it any more verbose with an explicit `OperationMetricsName` which is the same across all of the tests.

Reply:
I was under the impression it was necessary - I removed! I agree it does not help with verbosity.

Reply:
I was under the impression `OperationMetricsName` was mandatory but I just removed it and it works! I agree it doesn't make things more verbose.